src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Ball();
+        Lottery ball2 = new Ball();
+        Lottery ball3 = new Ball();
```

Comment:
Creating multiple instances of 'Ball' is unnecessary if you can use only one instance to achieve the desired behavior. Consider using a single 'Lottery' instance and calling 'getRandomBall()' on it multiple times.